UrbanAirship.Android.Accengage.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
    <metadata>
       <id>urbanairship.android.accengage</id>
-      <version>12.2.0</version>
+      <version>12.2.3</version>
```

Comment:
Why is Android changing as part of an iOS ticket?

Reply:
From the airship.properties the Android version is already 12.2.3. I think, for some reason, the two files **UrbanAirship.Android.Accengage.nuspec**  and **src/AirshipBindings.Android.Accengage/AirshipBindings.Android.Accengage.csproj**  was not committed (I think that should be in this [PR](https://github.com/urbanairship/urbanairship-xamarin/pull/133/files). May be I should create another ticket for this one. What do you think?

Reply:
I think you're right, it should have been done in the previous PR. Since that was already released, we may need to release a patch with that. I think there must be something broken in our build, since the version numbers are supposed to be updated automatically. Let's see what Ryan wants to do.

Reply:
Just to capture our slack discussion here: Mouna will update the Accengage nuspec and csproj and test the Gradle `syncVersion` task that should have updated these versions automatically.

Reply:
The `syncVersion`  updated the `nuspec` file but does not update the `csproj` one.

Reply:
Yes, I think there is a problem with the syncVersion. If the new aar has already been downloaded, it won't update the `csproj`. Don't worry about it here. I'll make a ticket.